test/attr/dynamicReplacement.swift

```diff
@@ -50,14 +50,6 @@ extension K {
   func replacement_finalFunction() {}
 }
 
-extension undeclared { // expected-error{{use of undeclared type 'undeclared'}}
```

Comment:
Instead of deleting the test case, just update the expected-errors

Reply:
There aren't any meaningful errors in either file.  We just diagnose the extension now.

Reply:
My point still stands. We want to make sure we don’t regress and start crashing here. It’s testing a specific case where the extension has a missing type and a nested type is inside it.